libs/ledger-live-common/src/families/evm/__tests__/fixtures/common.fixtures.ts

```diff
@@ -83,7 +87,7 @@ export const makeAccount = (
     },
   };
 
-  return account;
+  return Object.freeze(account);
```

Comment:
Can you explain why we need all those Object.freeze ?

Reply:
Mutating an object that is going to be used by our methods can be very problematic as the execution of a test can change the object for the next tests.
Object.freeze make sure that you can't mutate that Object (which is a good thing with React), so everytime you're using it with a function that mutates it, you'll have to use a copy, and all the tests are now safe to always use the Object you intented.